llvm/test/Instrumentation/AddressSanitizer/SPIRV/instrument_private_address_space.ll

```diff
@@ -21,11 +21,15 @@ define spir_kernel void @kernel() #0 {
 entry:
   %p.i = alloca [4 x i32], align 4
   ; CHECK: %shadow_ptr = call i64 @__asan_mem_to_shadow(i64 %0, i32 0)
+  ; CHECK: call void @__asan_set_shadow_private(i64 %4, i64 2, i8 -15)
+  ; CHECK: call void @__asan_set_shadow_private(i64 %5, i64 1, i8 -13)
```

Comment:
```suggestion
  ; CHECK: call void @__asan_set_shadow_private(i64 %5, i64 1, i8 -13)
```
for my education, why does the last parameter changes?

Reply:
Hi @wenju-he, the first call is to poison left red zone, and -15 is magic number of left red zone. And second call is to poison right red zone, -13 is magic number of it.

Reply:
thank you @zhaomaosu